Updates to the curating the editor experience to include 6.1 & 6.2 items (#48294)
WordPress/gutenberg: docs/how-to-guides/curating-the-editor-experience.md

![Image of locking interface](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/Locking%20interface.png?raw=true)

> Keep in mind that you can apply locking options to blocks nested inside of a containing block by turning on the "Apply to all blocks inside" option. However, you cannot mass lock blocks otherwise.

Reviewer: I suggest adding in here that using this option in the UI automatically adds the lock attribute to all the nested blocks. 

Reading it as is, sounds like there is a special attribute you could also add via code which would apply it to all descendants. 

Maybe the "However, you cannot mass lock blocks otherwise" is meant to say just that but I'm not sure that conveys it.

Author: >Maybe the "However, you cannot mass lock blocks otherwise" is meant to say just that but I'm not sure that conveys it.

Yeah, that is what I'm trying to convey! Going to resolve for now but we can always iterate.

> Keep in mind that you can apply the locking options to blocks inside of the initial block you are locking by turning on the "Apply to all blocks inside" option but you cannot mass lock blocks otherwise. 

Reviewer 2: This is almost of a bit of a tongue-twister with all the mentions of "lock" and "block" in a single sentence. I definitely had to read it a couple of times to make sure I understood.

Not sure if this is a lot better, but just a rough go at it:

> Keep in mind that you can apply locking options to blocks nested inside of a containing block by turning on the "Apply to all blocks inside" option. However, you cannot mass lock blocks otherwise.

Author: Updated! Thank you for the rewrite option.

## Utilizing patterns

**Prioritize starter patterns for any post type**

> When a user creates new content, whether a page or post, they are met with an empty canvas. However, that experience can be improved thanks to the option to have patterns from a specific type prioritized upon creation of a new piece of content. The modal appears each time the user creates a new item when there are patterns on their website that declare support for the core/post-content block types. By default, WordPress does not include any of these patterns, so the modal will not appear without some of these post content patterns being added. 

Reviewer: ```suggestion
When a user creates new content, regardless of post type, they are met with an empty canvas. However, that experience can be improved thanks to the option to have patterns from a specific type prioritized upon creation of a new piece of content. The modal appears each time the user creates a new item when there are patterns on their website that declare support for the `core/post-content` block types. By default, WordPress does not include any of these patterns, so the modal will not appear without at least two of these post content patterns being added. 
```

> To opt into this, include core/post-content in the blockTypes for your pattern. [Here's an example of a pattern](https://gist.github.com/annezazu/ead4c4965345251ec999b716c0c84f32) that would appear when creating a new post. 

Reviewer: ```suggestion
To opt into this, include `core/post-content` in the Block Types for your pattern. From there you can control which post types the pattern should show up for via the Post Types option. [Here's an example of a pattern](https://gist.github.com/annezazu/ead4c4965345251ec999b716c0c84f32) that would appear when creating a new post. 
```

> Read more about this functionality in the [Page creation patterns in WordPress 6.0 dev note](https://make.wordpress.org/core/2022/05/03/page-creation-patterns-in-wordpress-6-0/) and note that WordPress 6.1 brought this functionality to all post types.  

Reviewer: ```suggestion
Read more about this functionality in the [Page creation patterns in WordPress 6.0 dev note](https://make.wordpress.org/core/2022/05/03/page-creation-patterns-in-wordpress-6-0/) and [note that WordPress 6.1 brought this functionality to all post types](https://make.wordpress.org/core/2022/10/10/miscellaneous-editor-changes-for-wordpress-6-1/#start-content-patterns-for-all-post-types).  
```
